src/hotspot/share/runtime/thread.cpp

```diff
@@ -2096,10 +1824,6 @@ void JavaThread::wait_for_object_deoptimization() {
   bool spin_wait = os::is_MP();
   do {
     set_thread_state(_thread_blocked);
-    // Check if _external_suspend was set in the previous loop iteration.
```

Comment:
Note that the comment for this method at line 1806 needs updating as it refers to a now deleted method.

Reply:
I removed that line, not sure if the comment needs additional fixing.

Reply:
Thanks!